app/ui/archiveTile.js

```diff
@@ -140,6 +142,26 @@ function archiveDetails(translate, archive) {
 }
 
 module.exports = function(state, emit, archive) {
+  const copyOrShare =
+    typeof window.Android !== 'object'
```

Comment:
we should standardize how we check for android. I used `utils/browserName` somewhere else to do the same thing. I don't really care how we do it but we should probably wrap it in a function.

Reply:
Good call. I switched to browserName.